Makefile

```diff
@@ -140,7 +140,7 @@ TESTS = $(basename $(TEST_SRC))
 test: $(TESTS)
 
 $(TESTS) : %: %.scm
-	./cyclone -I . $<
+	./cyclone -I libs -COPT '-Iinclude' -CLNK '-L.' $<
```

Comment:
It is unclear to me what the propose of `-I .` is/was. Without explicitly specifying the `./libs` subdirectory as an argument to `-I` I still get the error `Error: Unable to open file: "/usr/lib/cyclone/cyclone/test.sld"` when running `make test`. Maybe I also misunderstood the propose of `-I .` in the first place.

Reply:
This is an excellent change, thank you!

Regarding `-I`, the issue is that we need to search `libs` in order to find `(cyclone test)` . The library cannot be found within `.` because `libs` is not part of the library paths. 

This is not a problem when everything is installed but we can't have a `cyclone` directory within our local source tree because the name would conflict with the `cyclone` executable of the same name. I understand there are other possible solutions but this is how we chose to solve the problem. Does that make sense?